src/CallbackHandler.php

```diff
@@ -84,15 +78,9 @@ public function getCallback()
     public function call(array $args = array())
     {
         $callback = $this->getCallback();
-
-        // Minor performance tweak, if the callback gets called more than once
-        if (!isset(static::$isPhp54)) {
-            static::$isPhp54 = version_compare(PHP_VERSION, '5.4.0rc1', '>=');
-        }
-
         $argCount = count($args);
 
-        if (static::$isPhp54 && is_string($callback)) {
+        if (is_string($callback)) {
```

Comment:
This should be wrong as the operator was `&&`, thus the whole condition shouldn't be required without it being 5.4